src/batch-client/index.ts

```diff
@@ -503,6 +505,14 @@ export class ZimbraBatchClient {
 			}
 		}).then(res => normalize(CalendarItemHitInfo)(res));
 
+	public sendMessage = (body: SendMessageInput) =>
+		this.jsonRequest({
+			name: 'SendMsg',
+			body: {
+				...denormalize(SendMessageInfo)(body)
```

Comment:
@pl12133  said we shouldn't need to use `...` with denormalize and normalize, and is trying to get rid of their use